src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -244,7 +236,7 @@ def extract_strings(self) -> Mapping[UseCaseID, Mapping[OrgId, Set[str]]]:
                         "use_case_id": use_case_id,
                         "org_id": org_id,
                         "metric_name": metric_name,
-                        "invalid_tags": invalid_strs,
+                        "tags": tags,
```

Comment:
We lose the capability to log the exact tags which are invalid with this change. Which is fine in my opinion. Someone can look at the tags list and find it out. The interface should only tell us whether tags are valid or not.